charts/osm/crds/multiclusterservice.yaml

```diff
@@ -43,6 +43,22 @@ spec:
               properties:
                 serviceAccount:
                   type: string
+                ports:
+                  description: The list of ports that are exposed by this service.
+                  type: array
+                  items:
+                    type: object
+                    required:
+                      - port
+                      - protocol
+                    properties:
+                      port: 
+                        description: The port that will be exposed by this service.
+                        type: integer
```

Comment:
please add a minimum and maximum value for the post like this : https://github.com/openservicemesh/osm/blob/bc441bae296c306f37515387f8a92d8f38660657/charts/osm/crds/policy.yaml#L89

Reply:
done!